nixops/resources/ec2_security_group.py

```diff
@@ -167,7 +167,9 @@ def create(self, defn, check, allow_reboot, allow_recreate):
                 grp = self.get_security_group()
             for rule in new_rules:
                 if len(rule) == 4:
+                    # VVVV SOMETHING HERE TO RETRY
                     grp.authorize(ip_protocol=rule[0], from_port=rule[1], to_port=rule[2], cidr_ip=rule[3])
+                    # ^^^^ SOMETHING HERE TO RETRY
```

Comment:
So this fails, because it could not find the previous security group. It might take some time before it is created, so we should probably retry this a few times.